docs/js/validate.js

```diff
@@ -0,0 +1,215 @@
+const fullName             = document.getElementById( 'name' );
+const email                = document.getElementById( 'email' );
+const helpDescription      = document.getElementById( 'help-comment' );
+const form                 = document.querySelector( 'form' );
+const errorSummary         = document.getElementById('error-summary');
+const errorSummaryMessages = document.querySelector( '#error-summary p' );
+const errorMessageHeading  = document.querySelector( '#error-summary h3' );
+const errors               = { name: '' , email: '' , 'help-comment': ''};
+
+/**
+ * Add event listener when form submitted
+ */
+form.addEventListener( 'submit', function( event ) {
+	if ( validateForm() ) {
```

Comment:
What's the point of this?

Reply:
woops forgot to get rid of it after a refactor